README.md

```diff
@@ -1,2 +1,44 @@
-# ActiveLearningAPI
-API for active learning
+# Active Learning API
+
+
+We use [FastAPI](https://fastapi.tiangolo.com/).
+
+# Documentation
+
+Documentation can be viewed at `/docs`. This is automatically generated from the code.
```

Comment:
How do I automatically generate them?

Reply:
ill add a comment to the code,

basically, you have to run the api, then go to http://localhost/docs or wherever it is running